spec/spec_helper_local.rb

```diff
@@ -33,6 +33,19 @@ def with_openbsd_facts
   end
 end
 
+def with_freebsd_facts
+  # operatingsystemmajrelease is too broad
```

Comment:
do we mock stuff here that is totally different in the real world, depending on the freebsd trouble? does it bring us into trouble?

Reply:
Not quite sure I follow?

Reply:
are the facts we simulate here used? If so, we should probably simulate them correctly, for all supported freebsd versions (or rely on rspec-puppet-facts)

Reply:
ps - reformatted this one. I think they are (like the other OSes in this file) valid, but not the only possible value.
For example, we support CentOS 6, but only supply facts for 7.